indra/llcommon/llsdserialize.h

```diff
@@ -858,14 +858,19 @@ class LL_COMMON_API LLUZipHelper : public LLRefCount
         ZR_SIZE_ERROR,
         ZR_DATA_ERROR,
         ZR_PARSE_ERROR,
+		ZR_BUFFER_ERROR,
+		ZR_VERSION_ERROR
```

Comment:
Please use indentation consistent with the existing entries.

Reply:
Second merge removed these so solved!